### PR TITLE
feat(web-app): #499 next/image support for mdx

### LINKED ISSUE #499: Add MDX image support using Next.js image component
- need to render static markdown images from mdx

`alt`

### PR DESCRIPTION
**Pull request by francinelucca**
Closes #499 

#### Testing / reviewing

Add file `test.mdx` under `services/web-app/pages` containing:

```
import carbonImg from "../public/carbon.png"

<Image alt="alt" src={carbonImg} />

![alt](/carbon.png)
```

Make sure you're able to access "http://localhost:3000/test", that it renders both images and that it resizes as the screen width changes

### CODE REVIEW

**Review by alisonjoseph — CHANGES_REQUESTED** [review]
Yay! This is working awesome 🥳 , lets pull the image out into its own component just to keep things cleaner.

**Comment by francinelucca** [comment]
> Yay! This is working awesome 🥳 , lets pull the image out into its own component just to keep things cleaner.

Done!

File: services/web-app/components/mdx/components.js [hunk 1]
```diff
@@ -4,9 +4,12 @@
  * This source code is licensed under the Apache-2.0 license found in the
  * LICENSE file in the root directory of this source tree.
  */
+
 import Link from '@/components/link'
 import { Blockquote, H1, H2, H3, H4, H5, H6, LI, OL, P, UL } from '@/components/markdown'
 
+import Image from '../markdown/image'
```
Comment (alisonjoseph) [inline services/web-app/components/mdx/components.js:11]:
```suggestion
import Image from '@/components/markdown/image'
```
super small nitpick

**Review by alisonjoseph — DISMISSED** [review]
🔥

**Review by andreancardona — APPROVED** [review]
Looks good! There is that alt prop warning but I know not sure if that's needed since we have it in the Image component - thoughts?

**Review by alisonjoseph — APPROVED** [review]

**Comment by alisonjoseph** [comment]
I think we can safely ignore that warning @andreancardona 👍

**Pull request merged**